build/parser.mjs

```
@@ -1038,20 +1038,38 @@ class Parser {
    * @param {Array<module:@wfcd/relics.TitaniaRelic>} relics relic array to search
    */
   addRelics(item, relics) {
-    if (item.type !== 'Relic') return;
+    const hasRelicDrops =
+      item.components && item.components.some((c) => c.drops?.some((d) => d.location?.includes('Relic')));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)